### PR TITLE
TATB - 37 : Redesign Quiz Test Component

### PR DESCRIPTION
**Pull request by PrabhatJTDeveloper**

### CODE REVIEW

File: src/app/component/quiz/quiz.component.scss [hunk 1]
```diff
@@ -1,57 +1,73 @@
...
+  }
 }
-.card-body-layout {
-  padding: 20px 60px 60px;
-  margin: 8px;
-  display: inline-block;
-  border-radius: 5px;
+
+.que_txt {
+  padding: 15px 15px;
```
Comment (charmijtdev) [inline src/app/component/quiz/quiz.component.scss]: can we use p-3 ?

File: src/app/component/quiz/quiz.component.scss [hunk 2]
```diff
@@ -1,57 +1,73 @@
...
+  }
 }
 
-.form-check {
-  margin: 20px auto;
-  font-family: Roboto;
+.option {
+  background: rgb(222, 239, 255);
+  border: 1px solid #84c5fe;
+  padding: 8px 15px;
```
Comment (charmijtdev) [inline src/app/component/quiz/quiz.component.scss]: same as can we manage with p class ?

File: src/app/component/quiz/quiz.component.scss [hunk 3]
```diff
@@ -1,57 +1,73 @@
...
+  font-size: 17px;
+  cursor: pointer;
+  margin-top: 15px;
```
Comment (charmijtdev) [inline src/app/component/quiz/quiz.component.scss]: please use mt-4

**Review by charmijtdev — CHANGES_REQUESTED** [review]

File: src/app/component/quiz/quiz.component.scss (hunk 1 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/quiz/quiz.component.scss]: yes, i changed it

File: src/app/component/quiz/quiz.component.scss (hunk 2 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/quiz/quiz.component.scss]: yes, i changed it.

File: src/app/component/quiz/quiz.component.scss (hunk 3 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/quiz/quiz.component.scss]: yes, i changed it.

File: src/app/component/quiz/quiz.component.html [hunk 4]
```diff
@@ -1,108 +1,145 @@
...
+              *ngFor="let arrayloop of formArray.controls; let i = index"
+            >
+              <ng-container [formGroupName]="i">
+                <div class="m-3">
+                  <div class="card quiz_card h-auto">
+                    <div
+                      class="card-header header_txt py-0 px-3 d-flex align-items-center justify-content-between position-relative"
+                    >
+                      <div class="header_title user-select-none">
+                        {{ [this.selectedQuizType] }}
```
Comment (charmijtdev) [inline src/app/component/quiz/quiz.component.html]: why used like this and never use this inside of html file

**Review by charmijtdev — CHANGES_REQUESTED** [review]

File: src/app/component/quiz/quiz.component.html (hunk 4 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/quiz/quiz.component.html]: yes, i changed it.

**Review by charmijtdev — APPROVED** [review]

**Pull request merged**